src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/DocumentMapper.java

```diff
@@ -36,7 +37,7 @@ public static ScannedDocument mapDocument(Document document) {
                 document.type,
                 document.subtype,
                 getLocalDateTime(document.scannedAt),
-                new CcdDocument(document.url),
+                new CcdDocument(StringUtils.join(dmApiUrl, "/", document.uuid)),
```

Comment:
Same here, `String.join`

Reply:
you could append context(/documents) and uuid to the dmurl.